### ISSUE TITLE
in-place operations raise error

### ISSUE THREAD

**Issue by daurer** [FilipeMaia/afnumpy#11]
``` python
a = afnumpy.ones((2,2))
a *= 1
```

raises an `AttributeError`:

``` bash
AttributeError                            Traceback (most recent call last)
<ipython-input-29-a0b68920a011> in <module>()
----> 1 b -= 0

/Library/Python/2.7/site-packages/afnumpy/decorators.pyc in wrapper(*args, **kws)
     27             args = bcast_args
     28 
---> 29         ret = func(*args, **kws)
     30         if len(ret.shape) == 0:
     31             return ret[()]

/Library/Python/2.7/site-packages/afnumpy/multiarray.pyc in __isub__(self, other)
    156     @iufunc
    157     def __isub__(self, other):
--> 158         afnumpy.subtract(self, pu.raw(other), out=self)
    159         return self
    160 

/Library/Python/2.7/site-packages/afnumpy/decorators.pyc in wrapper(*args, **kws)
     10         if out is not None:
     11             if(out.ndim):
---> 12                 out[:] = ret
     13             else:
     14                 out[()] = ret

/Library/Python/2.7/site-packages/afnumpy/multiarray.pyc in __setitem__(self, idx, value)
    404         else:
    405             raise NotImplementedError('values must be a afnumpy.ndarray')
--> 406         self.reshape(input_shape).d_array[idx] = value
    407         # This is a hack to be able to make it look like arrays with stride[0] > 1 can still be views
    408         # In practise right now it only applies to ndarray.real and ndarray.imag

/Library/Python/2.7/site-packages/afnumpy/__init__.pyc in inplace_setitem(self, key, val)
     13     try:
     14         n_dims = self.numdims()
---> 15         if (arrayfire.util._is_number(val)):
     16             tdims = arrayfire.array._get_assign_dims(key, self.dims())
     17             other_arr = arrayfire.array.constant_array(val, tdims[0], tdims[1], tdims[2], tdims[3], self.type())

AttributeError: 'module' object has no attribute '_is_number'
```

Same happens for `+=` , `-=` and `/=`.

**Comment by FilipeMaia** [comment]
What version of arrayfire do you have?

**Comment by daurer** [comment]
I was using arrayfire-python 3.0, problem is fixed with 3.2
